src/components/HistoricalRecordSelector/index.tsx

```
@@ -0,0 +1,544 @@
+                              : ""}
+                          </p>
+                          {encounter.organizations &&
+                            encounter.organizations.map((org) => (
+                              <Badge
+                                variant="outline"
+                                key={org.id}
+                                className="ml-2"
+                              >
+                                {org.name}
+                              </Badge>
+                            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)